### PR TITLE
Created initial folder for Arrays with a program on array to start with.

### PR DESCRIPTION
**Pull request by omkarlanghe**
- Added initial sub folder called array
- Program contribution based on array to find sub array with given sum.
- Updated README.md for the same.

### CODE REVIEW

**Comment by omkarlanghe** [comment]
Can anyone help me to figure out what I need to modify in order to ensure successful build ? Also, I have checked my code format using cpplint. Still I am not able to figure it out what might be wrong.

I have added new folder known as **array** inside C++ arrays section. This is the reason I feel my build is failing. But I am not sure.

**Comment by omkarlanghe** [comment]
Solved. I renamed folder name and .cpp file name.

File: C++/README.md [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
 
 ## Data Structures
-_add list here_
-1.Arrays.
-2.Linked List.
-3.Stack.
-4.Queue.
-5.Graph
+### Arrays
+* [Contiguous Sub Array with Given Sum](Arrays/SubArrayWithGivenSum.cpp)
```
Comment (ashwanisng) [inline C++/README.md:12]: Try this -
[Contiguous Sub Array with Given Sum](arrays/SubArrayWithGivenSum.cpp)

Comment (omkarlanghe) [reply C++/README.md:12]: @ashwanisng I didn't get. Can you please elaborate what should I do ?

**Comment by HarshCasper** [comment]
Reviews and Merged 🙌

Thanks for your contribution, @omkarlanghe 🎉🤗

**Pull request merged**